docs/components/Navigation.tsx

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable -- Lots of Nextra magic. */
```

Comment:
this feels like a prettttttty big hammer. is it worth it to list out all the rules instead?

Reply:
Oh, yup, good call.